src/styles/index.scss

```diff
@@ -1,3 +1,203 @@
+$star-number: 5;
+$color-minor: #616070;
+$color-accent: #00acdc;
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  background-color: #fff;
+  color: #060b35;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+
+  width: 200px;
+  height: 408px;
```

Comment:
Consider avoiding static heights, the height should resize accordingly to the content inside

Reply:
Removed the static height. 
Can the width be specified?